Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift

```diff
@@ -37,7 +37,17 @@ public enum PaymentMethodType: RawRepresentable, Hashable, Codable {
     case econtextStores
     case econtextATM
     case econtextOnline
+    case boletoBancario
+    @available(
+        *,
+        deprecated,
+        renamed: "boletoBancarioSantander",
```

Comment:
Let's guide merchants regarding the right tx variant. In case of Santander, let's suggest to use  boletoBancarioSantander otherwise to the default boletoBancario.